src/mavlink-router/autolog.cpp

```diff
@@ -73,6 +73,7 @@ void AutoLog::stop()
 bool AutoLog::start()
 {
     _system_id = LOG_ENDPOINT_SYSTEM_ID;
+    _add_sys_comp_id(LOG_ENDPOINT_SYSTEM_ID << 8);
```

Comment:
do we need it here? It's only purpose is to "wait for heartbeat to decide which logger to start". It will receive a heartbeat from the flight stack since that's a broadcast.

In `write_msg()` when we call `_logger->start()` we will call LogEndpoint::start() which will do everything needed.